data/plugins/mailjet.ts

```diff
@@ -11,6 +11,42 @@ export const Mailjet: Plugin = {
   otherTypes: [],
   category: "email marketing",
   badge: "",
-  showLink: false,
+  showLink: true,
   packageName: "@grouparoo/mailjet",
+  useCases: {
+    paragraphOne:
+      "Grouparoo is open source data infrastructure to keep customer data in-sync between your data warehouse and 3rd party tools such as Mailjet.",
+    paragraphTwo:
+      "Grouparoo pulls customer data in real-time from all of your data sources and warehouse and keeps that data synced to Mailjet.",
+  },
+  useCasesList: [
+    "Automatically create Contacts as new users sign up in your core product",
+    "Trigger email campaigns based on dynamic Audiences",
+  ],
+  dataModelTitle: "An Overview of Mailjet’s Data Model and API",
+  dataModelDescription: [
+    "Trying to figure out how to integrate with Mailjet? Here’s an overview for how Mailjet defines customers.",
+    "The core object in Mailjet is a Contact. A Contact represents an individual person that lives on a Contact list. Contacts have one required attribute, which is their email address.",
+    "Mailjet calls attributes of a Contact a Contact Property. Mailjet pre-defines a number of Contact Properties such as <code>firstname</code> and <code>country</code>, but you can define as many Contact Properties as you'd like in the Mailjet app.",
```

Comment:
Using `app` might imply a Grouparoo app - maybe `Mailjet website` or just `in Mailjet`?